.github/data/auto-review-comment.yml

```
@@ -0,0 +1,8 @@
+- prependMsg: "🗯️ [pull-request-auto-reviewer]:"
```

```suggestion
- prependMsg: "Pull request auto-reviewer"
```

Let’s do it less robotic :)